e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -192,6 +198,51 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
     cy.findByText("Orders");
   });
+
+  it(
+    "should allow settings click behavior on boolean fields (metabase#18067)",
+    { tags: "@external" },
+    () => {
+      const dialect = "mysql";
+      const TEST_TABLE = "many_data_types";
+      resetTestTable({ type: dialect, table: TEST_TABLE });
+      restore(`${dialect}-writable`);
+      cy.signInAsAdmin();
+      resyncDatabase({
+        dbId: WRITABLE_DB_ID,
+        tableName: TEST_TABLE,
+        tableAlias: "testTable",
+      });
+
+      cy.get("@testTable").then(testTable => {
+        const dashboardDetails = {
+          name: "18067 dashboard",
+        };
+        const questionDetails = {
+          name: "18067 question",
+          database: WRITABLE_DB_ID,
+          query: { "source-table": testTable.id },
+        };
+        cy.createQuestionAndDashboard({
+          dashboardDetails,
+          questionDetails,
+        }).then(({ body: { dashboard_id } }) => {
+          visitDashboard(dashboard_id);
+        });
+      });
+
+      editDashboard();
+
+      cy.log('Select "click behavior" option');
+      showDashboardCardActions();
+      cy.findByTestId("dashboardcard-actions-panel").icon("click").click();
```

Comment:
I think this could be clearer, maybe adding another util in `e2e/support/helpers/e2e-dashboard-helpers.js` would do that. But I don't want to think about the util API right now, so I'm using inline commands for now.